PhysicsTools/NanoAOD/python/nano_cff.py

```diff
@@ -28,6 +28,7 @@
 from PhysicsTools.NanoAOD.fsrPhotons_cff import *
 from PhysicsTools.NanoAOD.softActivity_cff import *
 from PhysicsTools.NanoAOD.l1trig_cff import *
+from PhysicsTools.NanoAOD.custom_btv_cff import *
```

Comment:
I don't think you need the import from, as it is present in autoNano

Reply:
I'm confused, are you going to remove the import ?

Reply:
I think we can remove the import from the `nano_cff.py` but this stays in autoNano